lib/new_relic/agent/datastores/mongo/obfuscator.rb

```diff
@@ -1,43 +0,0 @@
-# This file is distributed under New Relic's license terms.
```

Comment:
Looks like this change (`obfuscator.rb` -> `nosql_obfuscator.rb`) is appearing as a deletion and an addition rather than a rename. Hopefully the new file hasn't lost the git history of changes made to it.

Reply:
hmm that's a good point. if github treats it as an add/remove, it probably will lose the history prior to the move.